2 - ListsLogicAndIterations/Code/Lists.cs

```diff
@@ -7,27 +7,50 @@ public static class Lists
 	{
 		public static List<decimal> BuildList(decimal first, decimal second)
 		{
-			throw new NotImplementedException();
+            List<decimal> newlist = new List<decimal> { first, second };
+            return newlist;
+
+            //throw new NotImplementedException();
 		}
 
 		public static decimal GetValue(List<decimal> list, int position)
 		{
-			throw new NotImplementedException();
+            try
+            {
+                return list[position];
+            }
+            catch (Exception)
+            {
+                return 0.0M;
+            }
```

Comment:
Though you can handle the exception, it would have been preferred to check how many items exist in the list and then see if the position is above or equal to zero and not higher than the amount of items in the list.